js/arena.js

```diff
@@ -0,0 +1,51 @@
+export default class Arena {
+    constructor(name){
+      this.name = name.charAt(0).toUpperCase() + name.substring(1).toLowerCase();
+       this.gladiators = []
+    }
+  
+    addGladiator(gladiator){
+      if(this.gladiators.length < 2){
+        this.gladiators.push(gladiator)
+      } 
+      else {
+        console.log("never have more than 2 gladiators in it at a time")
+       }
+    }
+
+    fight(){
+      const [gladiator1, gladiator2] = this.gladiators
+      const [weapon1, weapon2] = [gladiator1.weapon, gladiator2.weapon]
+      console.log(weapon1)
+      console.log(weapon2)
+        if (weapon1 === 'Trident' && weapon2 ==='Spear'){
```

Comment:
Missing the case for draw, when both gladiators have the same weapon. The gladiators array should be empty